core/src/test/java/org/opensearch/sql/expression/datetime/ConvertTZTest.java

```diff
@@ -20,111 +22,115 @@ class ConvertTZTest extends ExpressionTestBase {
 
   @Test
   public void invalidDate() {
-    FunctionExpression expr = DSL.convert_tz(DSL.datetime(
-            DSL.literal("2021-04-31 10:00:00")),
-        DSL.literal("+00:00"),
-        DSL.literal("+00:00"));
-    assertEquals(DATETIME, expr.type());
-    assertEquals(nullValue(), expr.valueOf());
+    assertThrows(SemanticCheckException.class,
```

Comment:
exception should be thrown on `expr.valueOf()` only, please reformat the test. You need to update the last assert only.

Reply:
Good catch, I have updated the tests in https://github.com/Bit-Quill/opensearch-project-sql/commit/f6edc6be62abbe3a3974665e8d51c515584234e0